brainiak/reprsimil/brsa.py

```diff
@@ -1226,7 +2090,98 @@ def _fit_diagV_GP(
                 break
 
         return current_vec_U_chlsk_l, current_a1, current_logSNR2,\
-            current_GP, X0
+            current_GP, X_res
+
+    def _fit_null(self, Y, X_base, scan_onsets=None):
```

Comment:
I have not verified that such calling would replace `_fit_null`. But there is no urgency for this.